src/generator.ts

```diff
@@ -217,7 +217,7 @@ export class Generator {
    * @return {string}
    */
   private sanitize(str: string, countryCode:string): string {
-    const latinWord: string = this.replaceChar(str).toLowerCase().replace(/\(|\)|\.|'|\//gi, '').replace(/ +|-|'/gi, ' ');
+    const latinWord: string = this.replaceChar(str).toLowerCase().replace(/\(|\)|\.|"|\//gi, '').replace(/ +|-|'/gi, ' ');
```

Comment:
i dont understand why you put a double quote here ?

Reply:
Yes I should have explained. It is to keep the exact same regex as in the AWS lambda: see https://gitlab.com/tictactrip/id-generator/-/blob/master/utils.js line 54